styleguide/source/_patterns/02-organisms/00-global/01-hub-header/hub-header.md

```diff
@@ -0,0 +1,5 @@
+---
+el: ".hub-header"
+title: "Hub Header"
+---
+The Hub Header is present on hub landing pages and consists of the AMA logo, a link to the AMA homepage, a title, a CTA button, CTA text, and regular text.
```

Comment:
Well technically now it doesn't have secondary text, just a phone number, right?